### PR TITLE
fix #7762 chore(nimbus): Add python script to aide in targeting expression testing (mobile)

### LINKED ISSUE #7762: Write targeting test tool
There are times when we need to test a specific targeting string against a specific context. There is currently no way to do this against the SDK. We should write a simple tool that follows the setup procedures that the `test_mobile_integration.py` integration test does, but allows us to specify a `context`, `custom_attributes` and a `targeting` expression.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2807)

### PR DESCRIPTION
**Pull request by jrbenny35**
Because
* Testing targeting expressions can sometimes be quite cumbersome to do locally

This commit
* Adds a script to allow a user to input a custom context and targeting expression and see how they evaluate.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -174,7 +174,7 @@ integration_shell:
 	$(COMPOSE_INTEGRATION) run firefox bash
 
 integration_sdk_shell: build_prod
-	$(COMPOSE_INTEGRATION) run rust-sdk bash
+	PYTHONPATH=/application-services/components/nimbus/src $(COMPOSE_INTEGRATION) run rust-sdk bash
```
Comment (yashika-khurana) [inline Makefile:177]: I think we should store the path in some variable and then use it here

**Review by yashika-khurana — CHANGES_REQUESTED** [review]
Let me message you on slack, I am trying to do these step

File: README.md [hunk 2]
```diff
@@ -427,6 +427,22 @@ An example using PYTEST_ARGS to run one test.
...
+
+This builds and sets up the mobile sdk for use in testing.
+
+Navigate to `app/tests/tools`
+
+To test a targeting expression, first add an app context named `app_context.json` to the `app/tests/tools` directory.
+
+You can then invoke the script with the `--targeting-string` flag:
+```bash
+python sdk_eval_check.py --targeting "(app_version|versionCompare('106.*') <= 0) && (is_already_enrolled)"
```
Comment (yashika-khurana) [inline README.md:440]: shouldn't be this `--targeting-string .........`

**Review by jaredlockhart — APPROVED** [review]
Wow.  Wow wow wow.  Wow.  This is VERY cool.  Just tested locally, works perfectly, documentation worked perfectly and is clear.  Very very cool.  I love it.  Ticket is written clearly, marked up correctly, code is very clear, documentation is clear.  This is super duper handy, 11/10 A++ 👏 👏 👏 👏 👏 

wow

**Review by yashika-khurana — APPROVED** [review]
Awesome looks perfect, all instructions are very clear. thanks, @jrbenny35 👏

**Comment by jrbenny35** [comment]
Thank you both!

**Pull request merged**